agent/agent.go

```diff
@@ -237,7 +237,7 @@ func NewAgent(options ...Option) (*Agent, error) {
 		} else {
 			agent.logger.Println("API key not found, agent can't be started")
 			return nil, errors.New(fmt.Sprintf("There was a problem initializing Scope.\n"+
-				"Check the agent logs at %s for more information.\n", agent.recorderFilename))
+				"Check the agent logs at %s for more information.\n\nScope DSN not found, agent can't be started", agent.recorderFilename))
```

Comment:
I think message was changed to:
```
SCOPE_DSN not defined, test results will not be sent to Scope.
Developing locally? Try out Scratchpad! https://app.scope.dev/local-dev/
```

Reply:
Yeah but that message was when we discuss to show it always, according to @fermayo now this message will show only if the debug flag is true. So the marketing thing of the second line makes no sense to me.

Reply:
OK, then we need to synchronize each other with this topic. 
Java agent does not have even `debug` flag. It has logger levels.

Reply:
Can we change the full message to just:
```
Scope DSN not found. Tests will run but no results will be reported to Scope. More info at https://docs.scope.dev
```
when debug is enabled.

Reply:
done